### ISSUE TITLE
Version 0.6.4 Breaks Test Explorer 

### ISSUE THREAD

**Issue by Tankanow** [kondratyev-nv/vscode-python-test-adapter#217]
### Version Info
```
Version: 1.51.1
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-11T01:11:34.018Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```

### Symptoms

After upgrading plugin to `0.6.4` and reloading VS Code, the Test Explorer View is empty and the Output `Python Test Adapter Log` does not exist at all.

#### Empty Test Explorer View

<img width="708" alt="Screen Shot 2020-12-09 at 3 25 11 PM" src="https://user-images.githubusercontent.com/2100107/101683303-c69ebe00-3a32-11eb-89eb-5410817d8300.png">

#### No Output

<img width="298" alt="Screen Shot 2020-12-09 at 3 26 11 PM" src="https://user-images.githubusercontent.com/2100107/101683387-e33af600-3a32-11eb-9dc9-39119f4a2e41.png">

### Relevant Settings

```
{
    "python.pythonPath": "<elided>/.pyenv/versions/3.8.5/bin/python",
    "pythonTestExplorer.testFramework": "pytest",
    "python.sortImports.path": "isort",
    "python.testing.cwd": "${workspaceFolder}",
    "python.testing.pytestEnabled": true,
    "python.testing.pytestArgs": [
        "tests",
        "-v",
        "-x"
    ],
   ...
}
```
### Remedy

For now, I can fix this by installing any version <= `0.6.3`.

### Other Notes

I'm not sure if this is related, but I _never_ use run/debug configurations in my coding workflow. I bring this up because `0.6.4` relates to `launch.json`.

**Comment by kondratyev-nv** [comment]
@Tankanow Thank you for reporting this! I've accidentally published invalid vsix file 😓 I've updated the extension with the same version now. Please, update (I'm not sure, but you might have to reinstall the extension) once again and let me know if the issue persists.

**Comment by Tankanow** [comment]
I still have the same issue. I even tried removing the directories from `~/.vscode/extensions`. 🤔

**Comment by kondratyev-nv** [comment]
I'll try to update dependencies and upload a new version today. Hope this will help.

**Comment by kondratyev-nv** [comment]
@Tankanow I've published a new version 0.6.5. Please, update and let me know if the issue persists.

**Comment by kondratyev-nv** [comment]
@Tankanow I'm closing this issue assuming that it's fixed. In case you have any issues feel free to reopen this or file another one.
